eslintconifg.json

```diff
@@ -0,0 +1,3269 @@
+yarn run v1.22.19
+$ /Users/digitalfabrik/projects/integreat-app/node_modules/.bin/eslint --print-config .eslintrc.js
+{
```

Comment:
This should not be commited, should it?

Reply:
Why do we need to commit this file? I thought it is just a translation from our `.eslintrc.js` to a `.json` file. And everything in this file is just duplicated. Is it used somewhere? Wo we really need both config files?

Reply:
Nope. This was added accidentally when I checked what rules excactly are emitted. Removed it again. Thank you for noting